.github/workflows/node.js.yml

```diff
@@ -5,25 +5,24 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
Maybe we should just use one version to simplify. It looks like this workflow was a copy-paste of a nodejs library where this was actually important, not for building our browser based lib.

Reply:
My idea was to test for maintained and LTS versions. 
If we aim to use simply one, should we use LTS?

Reply:
Yes, LTS sounds good.